<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerIhepStandGeometryModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerIhepStandGeometryModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>1089</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="4" column="0">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_Camera">
     <property name="text">
      <string>Camera</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QCheckBox" name="CheckBox_FixedReferenceCamera">
        <property name="text">
         <string>Fix Camera</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Beam's Eye View Camera</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="1" column="2">
            <widget class="QPushButton" name="PushButton_BevPlusX">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>+X</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="PushButton_BevMinusX">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-X</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="PushButton_BevPlusY">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>+Y</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="PushButton_BevMinusY">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>-Y</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="Label_BevOrientation">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Beam:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_CameraBeam">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLRTBeamNode</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Parameter Set</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_12">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_RtBeam">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLRTBeamNode</string>
            <string>vtkMRMLRTIonBeamNode</string>
            <string>vtkMRMLRTFixedIonBeamNode</string>
           </stringlist>
          </property>
          <property name="noneEnabled">
           <bool>true</bool>
          </property>
          <property name="addEnabled">
           <bool>false</bool>
          </property>
          <property name="removeEnabled">
           <bool>false</bool>
          </property>
          <property name="noneDisplay">
           <string>Please select a beam node</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Reference volume:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_ReferenceVolume">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLScalarVolumeNode</string>
           </stringlist>
          </property>
          <property name="noneEnabled">
           <bool>true</bool>
          </property>
          <property name="addEnabled">
           <bool>false</bool>
          </property>
          <property name="removeEnabled">
           <bool>false</bool>
          </property>
          <property name="noneDisplay">
           <string>Please select a node</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Target volume:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Fixed Beam:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_FixedBeam">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLRTFixedIonBeamNode</string>
            <string>vtkMRMLRTIonBeamNode</string>
           </stringlist>
          </property>
          <property name="showChildNodeTypes">
           <bool>false</bool>
          </property>
          <property name="noneEnabled">
           <bool>true</bool>
          </property>
          <property name="addEnabled">
           <bool>false</bool>
          </property>
          <property name="removeEnabled">
           <bool>false</bool>
          </property>
          <property name="noneDisplay">
           <string>Please select a fixed beam node</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Patient Beam:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="qMRMLSegmentSelectorWidget" name="SegmentSelectorWidget_TargetVolume"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Fixed DRR Beam:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_FixedDrrBeam"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="ctkCollapsibleButton" name="CollapsibleButton_Models">
     <property name="text">
      <string>Models</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="2" column="0">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_ModelsControl">
        <property name="title">
         <string>Software Position Control</string>
        </property>
        <property name="collapsed">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QCheckBox" name="CheckBox_RotatePatientHeadFeet">
             <property name="text">
              <string>Rotate Patient Head-Feet</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_PatientToTableTopTranslation">
             <property name="title">
              <string>Patient to Table Top Translation (mm)</string>
             </property>
             <property name="collapsed">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_13">
              <item row="0" column="0">
               <widget class="ctkCoordinatesWidget" name="CoordinatesWidget_PatientTableTopTranslation">
                <property name="minimum">
                 <double>-5000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>5000.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_TableTop">
             <property name="title">
              <string>Table Top Positions (mm)</string>
             </property>
             <property name="collapsed">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_15">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_14">
                <item row="1" column="1">
                 <widget class="ctkSliderWidget" name="SliderWidget_TableTopVerticalPositionOrigin">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>0.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>687.500000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Z Total</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="ctkSliderWidget" name="SliderWidget_TableTopVerticalPosition">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-999.899999999999977</double>
                  </property>
                  <property name="maximum">
                   <double>999.899999999999977</double>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="ctkSliderWidget" name="SliderWidget_TableTopStandLongitudinalPosition">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="maximum">
                   <double>875.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Y</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Z Origin</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Z Mirror</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="ctkSliderWidget" name="SliderWidget_TableTopVerticalPositionMiddle">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-687.500000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>687.500000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_13">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Z Middle</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="ctkSliderWidget" name="SliderWidget_TableTopVerticalPositionMirror">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-687.500000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>687.500000000000000</double>
                  </property>
                  <property name="value">
                   <double>0.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>X</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="ctkSliderWidget" name="SliderWidget_TableTopStandLateralPosition">
                  <property name="decimals">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <double>-209.375000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>209.375000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_PatientSupportRotationAngle">
             <property name="title">
              <string>Patient Support Rotation Angle (°)</string>
             </property>
             <property name="collapsed">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_16">
              <item row="0" column="0">
               <widget class="ctkSliderWidget" name="SliderWidget_PatientSupportRotationAngle">
                <property name="decimals">
                 <number>2</number>
                </property>
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>360.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="ctkCollapsibleButton" name="CollapsibleButton_TableTopAngles">
             <property name="text">
              <string>Table Top Angles (°)</string>
             </property>
             <property name="collapsed">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="0" column="1">
                 <widget class="ctkSliderWidget" name="SliderWidget_TableTopLongitudinalAngle">
                  <property name="minimum">
                   <double>-30.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>30.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="ctkSliderWidget" name="SliderWidget_TableTopLateralAngle">
                  <property name="minimum">
                   <double>-30.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>30.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Longitudinal</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Lateral</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="PushButton_ResetModelsInitialPosition">
             <property name="text">
              <string>Reset models to initial position</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_ModelsAndMarkupsVisibility">
        <property name="title">
         <string>Visibility</string>
        </property>
        <property name="collapsed">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="GroupBox">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="ctkCheckBox" name="CheckBox_MarkupsVisibility"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Models Visibility:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ctkCheckBox" name="CheckBox_ModelsVisibility"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Markups Visibility:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox">
        <property name="title">
         <string>Hardware Apparatus Position</string>
        </property>
        <property name="collapsed">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Control:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RadioButton_ModelsSoftwareControl">
          <property name="text">
           <string>Software</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroup_ModelsControl</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RadioButton_ModelsHardwareControl">
          <property name="text">
           <string>Hardware</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroup_ModelsControl</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="qSlicerBeamToStandTransformationWidget" name="SlicerWidget_BeamToStandTransformation"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header>ctkCoordinatesWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentSelectorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerBeamToStandTransformationWidget</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerBeamToStandTransformationWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerIhepStandGeometryModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>SegmentSelectorWidget_TargetVolume</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerIhepStandGeometryModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_FixedBeam</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerIhepStandGeometryModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_ReferenceVolume</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerIhepStandGeometryModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_RtBeam</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerIhepStandGeometryModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_FixedDrrBeam</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerIhepStandGeometryModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_CameraBeam</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="ButtonGroup_ModelsControl"/>
 </buttongroups>
</ui>
